testplan/testing/multitest/driver/base.py

```diff
@@ -337,34 +348,36 @@ def _setup_file_logger(self, path):
         When a file logger is finished with, _close_file_logger() should be
         called to close the opened file object and release the file handle.
         """
-        if self.file_logger is not None:
-            raise RuntimeError("File logger already exists")
-
-        formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
-        handler = logging.FileHandler(path)
-        handler.setFormatter(formatter)
-        logger = logging.getLogger("FileLogger_{}".format(self.cfg.name))
-        logger.addHandler(handler)
-        logger.setLevel(self.logger.getEffectiveLevel())
-        self.file_logger = logger
-        self.file_logger.propagate = False  # No console logs
+        if self._file_log_handler is not None:
+            raise self.logger.debug(
```

Comment:
raise logger.debug()? why debug() instead of raise?

Reply:
Similar code in TestRunner's `_config_file_logger` there is raise. But I think for a driver some times it will be restarted and so the no need to raise an exception but just log a message.

Reply:
after discussion we should raise here strictly. It's necessary to make sure that stop driver before restart and will not see this exception.